### PR TITLE
Add documentation for review alert features

### PR DESCRIPTION
**Pull request by Sengthay**

### CODE REVIEW

File: docs/xdr/features/detect/rules_catalog.md [hunk 1]
```diff
@@ -109,10 +109,30 @@ You can click on the name of a rule to display additional details, such as, but
...
 - Known False Positives
 - The actual detection logic (the pattern)
+- Alert filters
 - [Similarity strategy](../../investigate/alerts/#similarity-strategies) for the produced alerts
 
 ![rule details](/assets/operation_center/rules_catalog/rule_details2.png)
 
+### Alert filters
+
+In the section `Alert filters`, you can consult the list of alert filters that are currently enabled in the `Àctives` tab. The number of alert `muted` in the `last 30 days` is displayed for each alert filter to help you audit its effectiveness.
```
Comment (Carole-Besset) [inline docs/xdr/features/detect/rules_catalog.md:119]: @StephKll3c not sure if it has been done this way on back end side (last 30 days for telemetry)

Comment (Carole-Besset) [reply docs/xdr/features/detect/rules_catalog.md:119]: @Sengthay I think there is an accent on the uppercase A of "Actives" that should be removed

File: docs/xdr/features/detect/rules_catalog.md [hunk 2]
```diff
@@ -109,10 +109,30 @@ You can click on the name of a rule to display additional details, such as, but
...
+
+Click on the `arrow` to expand the alert filter and consult the exclusion pattern.
+
+![alert filters](/assets/operation_center/rules_catalog/alert_filters.png)
+
+The `Expired` tab list all the alert filters that had an `expiration date` set and whose date has been reached. Expired alert filters can be deleted if needed.
+
+### Rules with active alert filter
+
+To search for rules that have at least one active alert filter, use the search `Filters` and select the criteria `Rules with alert filters`.
```
Comment (Carole-Besset) [inline docs/xdr/features/detect/rules_catalog.md:129]:
```suggestion
To search for rules that have at least one active alert filter, use the search `Alert filter` and select the criteria `Rules with alert filters`.
```

File: docs/xdr/features/detect/rules_catalog.md [hunk 3]
```diff
@@ -109,10 +109,30 @@ You can click on the name of a rule to display additional details, such as, but
...
+All rules with active alert filters will be returned in the list.
+
+![search alert filters](/assets/operation_center/rules_catalog/search_alert_filters.png)
+
+You can also search for rules that no active alert filters with the other search criteria.
```
Comment (Carole-Besset) [inline docs/xdr/features/detect/rules_catalog.md:134]:
```suggestion
You can also search for rules that have no active alert filters with the other search criteria.
```

File: docs/xdr/features/detect/rules_catalog.md (hunk 1 above)
Comment (StephKll3c) [reply docs/xdr/features/detect/rules_catalog.md]: Indeed, there is no notion of 'in the last 30 days'.

**Review by StephKll3c — APPROVED** [review]

**Review by Carole-Besset — APPROVED** [review]

**Pull request merged**